drivers/include/vcnl40x0.h

```diff
@@ -0,0 +1,149 @@
+﻿/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_vcnl40x0 VCNL40X0
+ * @ingroup     drivers_sensors
+ * @brief       Device driver interface for the VCNL40X0 sensors family.
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the VCNL40X0 sensors family.
+ *
+ * @note        VCNL4010, VCNL4020 and VCNL4040 are supported.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef VCNL40X0_H
+#define VCNL40X0_H
+
+#include "saul.h"
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @brief Proximity measure rate
+ */
+enum {
+    VCNL40X0_PROXIMITY_RATE_2 = 0,               /**< Proximity measurement rate 2 */
+    VCNL40X0_PROXIMITY_RATE_4,                   /**< Proximity measurement rate 4 */
+    VCNL40X0_PROXIMITY_RATE_8,                   /**< Proximity measurement rate 8 */
+    VCNL40X0_PROXIMITY_RATE_16,                  /**< Proximity measurement rate 16 */
+    VCNL40X0_PROXIMITY_RATE_31,                  /**< Proximity measurement rate 31 */
```

Comment:
These are the values given in table 3, page 4 in the [datasheet](https://www.vishay.com/docs/83462/vcnl4010.pdf)